.github/workflows/python-package.yml

```diff
@@ -0,0 +1,201 @@
+name: Python package
+
+on:
+  push:
+    branches: [ "*" ]
```

Comment:
Unnecessary filter 😁 
```suggestion
```

Reply:
Agreed, but missing a {} in this case. I can fix later

Reply:
Fixed